src/ess/bifrost/data.py

```diff
@@ -41,7 +41,13 @@ def simulated_elastic_incoherent_with_phonon() -> str:
 
 
 def tof_lookup_table_simulation() -> str:
-    """Time-of-flight lookup table for the simulated BIFROST data."""
+    """Time-of-flight lookup table for the simulated BIFROST data.
+
+    This table was computed with `tof <https://github.com/scipp/tof>`_
+    using `Create a time-of-flight lookup table for BIFROST
+    <../../user-guide/bifrost/bifrost-make-tof-lookup-table.rst>`_
```

Comment:
Should the extension be html or rst (or even ipynb?)

Reply:
Always rst for internal links.